.npmrc

```diff
@@ -1,11 +1,10 @@
 engine-strict=true
 frozen-lockfile=true
 strict-peer-dependencies=true
+link-workspace-packages=true
 
 # Disable pnpm update notifications since we use corepack to install package managers
 update-notifier=false
-# Don't use new lockfile format because component governance does not yet support it
-use-lockfile-v6=false
```

Comment:
I don't think this did anything even before this change as I don't think pnpm 8 looked for this option. Regardless we are moving to format 9 now, so its out of date.